docs/api/checkouts.rst

```diff
@@ -785,7 +770,262 @@ Final response contains the full order information, now you can call the purchas
 
 .. code-block:: json
 
-    {"to": "do"}
+    {
+        "adjustments": [
+            {
+                "amount": 0,
+                "created_at": "2014-12-15T13:37:29+0000",
+                "description": "No tax (0%)",
+                "id": 205,
+                "type": "tax",
+                "locked": false,
+                "neutral": false,
+                "updated_at": "2014-12-15T13:37:29+0000"
+            },
+            {
+                "amount": 5000,
+                "created_at": "2014-12-15T14:30:41+0000",
+                "description": "FedEx World Shipping",
+                "id": 207,
+                "type": "shipping",
+                "locked": false,
+                "neutral": false,
+                "updated_at": "2014-12-15T14:30:41+0000"
+            },
+            {
+                "amount": -250,
+                "created_at": "2014-12-15T14:30:41+0000",
+                "description": "Christmas Sale for orders over 100 EUR.",
+                "id": 208,
+                "type": "promotion",
```

Comment:
"No more" promotion you mean :)
Doc hasn't been merged and it's already obsolete ^^